test/include/seqan3/test/tmp_filename.hpp

```diff
@@ -23,6 +23,23 @@
 
```

Comment:
Can you change the lines above to:

```cpp
#if defined(__APPLE__)
#include <unistd.h>
#elif defined(_WIN32)
#include <stdio.h>
#else  // other unix systems
#include <stdlib.h>
#endif
```

Reply:
OK. Investigating <stdio.h> vs <cstdio>: std:: or not?